DEVELOPMENT.md

```diff
@@ -114,6 +114,29 @@ We use [Cypress](https://www.cypress.io/) for integration tests and
 - To develop unit tests in watch mode, run: `yarn test:unit:watch`
 
 
+### Running multiple Upstream instances on the same machine
+
+At the moment this is only possible in development mode.
+
+```
+mkdir /Users/rudolfs/work/19000
+mkdir /Users/rudolfs/work/20000
+
+# Launch the first instance.
+
+RAD_HOME="/Users/rudolfs/work/19000" \
+RADICLE_UPSTREAM_PROXY_ARGS="--http-listen 127.0.0.1:19000 --peer-listen 0.0.0.0:19000" \
```

Comment:
I'd also just let the port be chosen as well
```suggestion
RADICLE_UPSTREAM_PROXY_ARGS="--http-listen 127.0.0.1:19000 --peer-listen 0.0.0.0:0" \
```

Reply:
If you do that it'll be harder to link them up if you want to test replication locally. You'll need to look at the logs to figure out on which port it bound. But I'll mention this possibility in the docs.

Reply:
bd7da77777fd597559eb25c23b8521b0bafba86b